web/frontend/src/components/slam-2d-render.vue

```diff
@@ -192,6 +326,28 @@ onUnmounted(() => {
   disposeScene();
 });
 
+watch(() => [props.destVector!.z, props.destVector!.x, props.destExists], async () => {
+  if (props.destVector && props.destExists) {
+    const marker = scene.getObjectByName('DestinationMarker') ?? await makeMarker(destMarkerUrl, 'DestinationMarker', 0.1);
```

Comment:
[nit] Mind please moving the `0.1` magic number into consts at the top of the file?